CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unversioned
 
 - Major: Added multi-channel searching to search dialog via keyboard shortcut. [Ctrl+Shift+F by default] (#3694)
+- Minor: Added option to highlight own messages in Highlights page (#3833)
```

Comment:
Specify that it's under the Users tab, and add a period at the end

Reply:
done in [51bc6c7](https://github.com/Chatterino/chatterino2/pull/3833/commits/51bc6c74f09c874b414e2e74ee9f137f2408d2df)